lib/manager/npm/post-update/index.ts

```diff
@@ -443,26 +443,12 @@ export async function getAdditionalFiles(
     hostType: 'npm',
   });
   for (const hostRule of npmHostRules) {
-    if (hostRule.token) {
-      if (hostRule.baseUrl) {
```

Comment:
Base url needs to be used too, as it can have sub-paths. I know it won't work to only use resolved host for yarn